src/model/model.py

```diff
@@ -9,15 +8,31 @@
 
 import sys
 from typing import List, Tuple, Optional
+from subprocess import CompletedProcess
+from typing import List
 from queue import Queue
 from os import path, listdir
+import subprocess
 
 from src.persistence.data_access import DataAccess
 from src.model.argument import Argument
 from src.model.fileinfo import FileInfo
 from src.model.argument_visitor import extract_arguments
 
 
+def run_program(command: str) -> CompletedProcess:
+    """
+    This method executes the given command.
+
+    Args:
+        command: contains the full path of the executable and after it, its arguments
+
+    Returns:
+        CompletedProcess: the result of the executed command
```

Comment:
Change the Return docstring

Reply:
What is the problem with this? What should I change?